### ISSUE TITLE
A few questions to consideration

### ISSUE THREAD

**Issue by MatejMazur** [MatejMazur/starter-webpack#1]
0) Who is the target group?
1) Is it the current config scheme ok?
2) Can we use the webpack-merge and Ramda's library or do we use custom function?
3) What is the minimal set for the basic usage? 
4) How many dependencies are needed?
5) What should look like the API of the extensions?
6) WIP...

**Comment by MatejMazur** [comment]
0. The devs who don't want to waste time with the starter kit.

**Comment by jbucaran** [comment]
0. Devs who like and appreciate webpack's power, but want a simplified UI and are willing to lose some of webpack's full featureset range.

**Comment by jbucaran** [comment]
@MatejMazur Updated.

**Comment by MatejMazur** [comment]
I've updated [comment](https://github.com/MatejMazur/starter-webpack/issues/1#issuecomment-323950307).

**Comment by jbucaran** [comment]
One more thing. The least amount of stuff you "make up" the better. Ideally, I should be able to easily migrate to webpack at some point or feel I've learned enough that I can "upgrade" to webpack.

But at the same time, this module should not exist solely as a stepping stone to use webpack. I should be able to continue using this module if I am content with what it provides and my as long as my use case allows.

In other words, let's try to be as transparent as possible.

**Comment by MatejMazur** [comment]
Updated.

**Comment by MatejMazur** [comment]
@jbucaran 

> 5. Maybe there should be no extensions.

How do you want to avoid the extensions? I mean that we don't need to support the extensions for public use but it still need to be done composable. So the extensions will be needed for internal manipulations.

**Comment by jbucaran** [comment]
@MatejMazur You are probably right! :)
